extractor/src/main/scala/com/digitalasset/extractor/json/JsonConverters.scala

```diff
@@ -63,6 +66,15 @@ object JsonConverters {
       JsonObject("Some" -> value.asJson).asJson
   }
 
+  implicit val mapEncoder: Encoder[HashMap[String, LedgerValue]] = m => {
```

Comment:
as discussed, please make it a "proper" JSON object, and enforce the DAML-LF map values to have string keys in protobuf